src/NzbDrone.Core/Download/Pending/PendingReleaseReason.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace NzbDrone.Core.Download.Pending
+{
+    public enum PendingReleaseReason
+    {
+        Delay = 0,
+        StoreAndForward = 1
```

Comment:
> StoreAndForward (horrible name)

If you have something better let's go for it. 😄 

> Delay & StoreAndForward items should be shown on the UI, dunno about Fallback, I think we should filter those.

Good point, I like that 👍